plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/selectablechangespanel/SelectionCallBack.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.ide.ext.git.client.compare.selectablechangespanel;
+
+/**
+ * Delegates select action into {@link SelectableChangesPanelPresenter}.
+ *
+ * @author Igor Vinokur
+ */
+public interface SelectionCallBack {
+
+  /** Is called when item check-box changed selection state. */
+  void onSelectionChanged();
```

Comment:
It is reasonable to pass item/state into the method

Reply:
done